lighthouse-core/audits/seo/font-size.js

```diff
@@ -177,9 +177,9 @@ class FontSize extends Audit {
       name: 'font-size',
       description: 'Document uses legible font sizes',
       failureDescription: 'Document doesn\'t use legible font sizes',
-      helpText: 'Font sizes less than 16px are too small to be legible and require mobile ' +
-      'visitors to “pinch to zoom” in order to read. Strive to have >75% of page text ≥16px. ' +
-      '[Learn more](https://developers.google.com/speed/docs/insights/UseLegibleFontSizes).',
+      helpText: 'Font sizes less than 12px are too small to be legible and require mobile ' +
+      'visitors to “pinch to zoom” in order to read. Strive to have >60% of page text ≥12px. ' +
+      '[Learn more](https://developers.google.com/web/fundamentals/design-and-ux/responsive/#optimize_text_for_reading).',
```

Comment:
@rviscomi FYI I updated the link to avoid the redirect + BTW do we have anything better we can link to?

Reply:
This is ok for now. Please work with @kaycebasques to get first-party LH SEO audit docs online and we can update this link when ready.